Resources/Prototypes/_RMC14/Entities/Clothing/Masks/masks.yml

```diff
@@ -68,6 +70,11 @@
     coverage: MOUTH
   - type: Item
     storedRotation: -90
+  - type: HideLayerClothing # TODO port pulled down sprite from SS13
+    slots:
+    - Hair
+    - HeadTop
+    - HeadSide
```

Comment:
Make this optional like helmets

Reply:
goes against parity cuz the coif hides hair if you have it equipped but if u really want it ill change it

Reply:
actually ill just put it in a seperate PR